### PR TITLE
Fix bug in document conversion

### PR DESCRIPTION
**Pull request by kvartet**
There are some links whose valid path cannot be found, you can use `cannot find` to search for them.

Although “githublink” is more elegant, they cannot be displayed properly on Github. Users will be confused when they use Github instead of the readthedocs to read documents.

Some problems need special attention:
1. Use Relative Paths. For example :
```
Github link, use :githublink:`mnist.py <examples/trials/mnist-tfv1/mnist.py>` , 
             instead of https://github.com/microsoft/nni/blob/master/examples/trials/mnist-tfv1/mnist.py
Other documents, `here <./Framework.rst>`__
```
Note : In the document `Release.rst`, absolute paths with `vx.x` should be used.
2. Update links in the document while renaming a file/folder.
3. rst documents do not support nesting. For example, when you want to bold a item with a link, like 
```
`**NNI** <https://github.com/microsoft/nni>`__ 
```
it won't work.
4. Use `.rst` instead of `.html` for filename extension
```
Use         `here <../TrainingService/PaiMode.rst>`__
Instead of  `here <../TrainingService/PaiMode.html>`__
```

### CODE REVIEW

File: docs/en_US/Assessor/CustomizeAssessor.rst [hunk 1]
```diff
@@ -60,6 +60,10 @@ The working directory of your assessor is ``<home>/nni-experiments/<experiment_i
 
 More detail example you could see:
 
+.. cannot find these links
+ :githublink:`medianstop-assessor <src/sdk/pynni/nni/medianstop_assessor>`
+ :githublink:`curvefitting-assessor <src/sdk/pynni/nni/curvefitting_assessor>`
+
 ..
 
    * :githublink:`medianstop-assessor <src/sdk/pynni/nni/medianstop_assessor>`
```
Comment (ultmaster) [inline docs/en_US/Assessor/CustomizeAssessor.rst:69]: nni/algorithms/hpo/medianstop_assessor

File: docs/en_US/CommunitySharings/AutoCompletion.rst [hunk 2]
```diff
@@ -27,6 +27,8 @@ Step 1. Download ``bash-completion``
 
 Here, {nni-version} should by replaced by the version of NNI, e.g., ``master``\ , ``v1.9``. You can also check the latest ``bash-completion`` script :githublink:`here <tools/bash-completion>`.
 
+.. cannot find :githublink:`here <tools/bash-completion>`.
```
Comment (ultmaster) [inline docs/en_US/CommunitySharings/AutoCompletion.rst:30]: Leave it here for now.

File: docs/en_US/CommunitySharings/NNI_AutoFeatureEng.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ NNI review article from Zhihu: :raw-html:`<an open source project with highly re
 
 The article is by a NNI user on Zhihu forum. In the article, Garvin had shared his experience on using NNI for Automatic Feature Engineering. We think this article is very useful for users who are interested in using NNI for feature engineering. With author's permission, we translated the original article into English.  
 
-**原文(source)**\ : `如何看待微软最新发布的AutoML平台NNI？By Garvin Li <https://www.zhihu.com/question/297982959/answer/964961829?utm_source=wechat_session&utm_medium=social&utm_oi=28812108627968&from=singlemessage&isappinstalled=0>`__
+**source**\ : `如何看待微软最新发布的AutoML平台NNI？By Garvin Li <https://www.zhihu.com/question/297982959/answer/964961829?utm_source=wechat_session&utm_medium=social&utm_oi=28812108627968&from=singlemessage&isappinstalled=0>`__
```
Comment (ultmaster) [inline docs/en_US/CommunitySharings/NNI_AutoFeatureEng.rst:10]: I think `\ :` is from conversion and not needed. If you see one of these, feel free to remove it.

File: docs/en_US/Compression/CustomizeCompressor.rst [hunk 4]
```diff
@@ -28,6 +28,8 @@ An implementation of ``weight masker`` may look like this:
            # mask = ...
            return {'weight_mask': mask}
 
+.. cannot find :githublink:`weight masker <src/sdk/pynni/nni/compression/pytorch/pruning/structured_pruning.py>`
```
Comment (ultmaster) [inline docs/en_US/Compression/CustomizeCompressor.rst:31]: Remove src/sdk/pynni/ part.

File: docs/en_US/TrialExample/Trials.rst [hunk 5]
```diff
@@ -80,6 +80,8 @@ To enable NNI API mode, you need to set useAnnotation to *false* and provide the
 
 You can refer to `here <../Tutorial/ExperimentConfig.rst>`__ for more information about how to set up experiment configurations.
 
+.. cannot find `here </sdk_reference.html>`__
```
Comment (ultmaster) [inline docs/en_US/TrialExample/Trials.rst:83]: /docs/en_US/sdk_reference.rst?

File: docs/en_US/Tuner/CustomizeAdvisor.rst [hunk 6]
```diff
@@ -17,7 +17,9 @@ If a user want to implement a customized Advisor, she/he only needs to:
        def __init__(self, ...):
            ...
 
-**2. Implement the methods with prefix ``handle_`` except ``handle_request``**.. You might find `docs </sdk_reference.html#nni.runtime.msg_dispatcher_base.MsgDispatcherBase>`__ for ``MsgDispatcherBase`` helpful.
+.. cannot find `docs </sdk_reference.html#nni.runtime.msg_dispatcher_base.MsgDispatcherBase>`__
```
Comment (ultmaster) [inline docs/en_US/Tuner/CustomizeAdvisor.rst:20]: Same here

File: docs/en_US/Tuner/CustomizeTuner.rst [hunk 7]
```diff
@@ -115,6 +115,11 @@ NNI needs to locate your customized tuner class and instantiate the class, so yo
 
 More detail example you could see:
 
+.. cannot find these link
```
Comment (ultmaster) [inline docs/en_US/Tuner/CustomizeTuner.rst:118]: nni/algorithms/hpo/xxx_tuner
nni/runtime/msg_dis...

File: docs/en_US/Tuner/BuiltinTuner.rst [hunk 8]
```diff
@@ -512,6 +512,8 @@ Note that the only acceptable types within the search space are ``layer_choice``
 
 **Suggested scenario**
 
+.. cannot find :githublink:`mnist-nas <examples/trials/mnist-nas>`
```
Comment (ultmaster) [inline docs/en_US/Tuner/BuiltinTuner.rst:515]: examples/nas/classic_nas

**Review by ultmaster — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**